templates/website/eslint.config.mjs

```diff
@@ -30,6 +30,9 @@ const eslintConfig = [
       ],
     },
   },
+  {
+    ignores: ['.next/'],
+  },
```

Comment:
this should not be necessary, I don't know why in our case we have to exclude this folder, but in `create-next-app` they don't.